src/execution/__tests__/sync-test.js

```diff
@@ -28,8 +28,8 @@ describe('Execute: synchronously when possible', () => {
         },
         asyncField: {
           type: GraphQLString,
-          async resolve(rootValue) {
-            return await rootValue;
+          resolve(rootValue) {
+            return Promise.resolve(rootValue);
```

Comment:
IMO it was very weird construct and explicit promise is a way more readable.

Reply:
I disagree in general, but it's not a big enough deal for me to block this diff.

Personally I much prefer async/await, but I don't love JavaScript, so my opinions are suspect.

Reply:
@mjmahone I reverted it back and just removed `await`, see 8cefd09bf71911648eb7def1b7d2f2feaa3dfb1f

Reply:
I reverted it back and just remove useless `await` so now it looks like:
```js
          async resolve(rootValue) {
            return rootValue;
          }
```